packages/web-lib/hooks/types.d.tsx

```diff
@@ -81,20 +81,25 @@ export type	TUseBalanceRes = {
 /* 🔵 - Yearn Finance **********************************************************
 ** Request, Response and helpers for the useBalances hook.
 ******************************************************************************/
+export type	TUseBalancesTokens = {
+	token?: string,
```

Comment:
I'm wondering if at least `token` should be required? Or can we have balances without tokens?